### PR TITLE
added BZE (beezee) blockchain details

### PR DESCRIPTION
**Pull request by faneaatiku**

### CODE REVIEW

**Comment by JeremyParish69** [comment]
Checks failed because bech32 prefix not registered to SLIP173 and images are not being uploaded to this repo.

**Comment by faneaatiku** [comment]
> SLIP173

I created the SLIP173 PR but I don't quite understand what's with the images? We no longer upload them here ? 
Thanks

File: beezee/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "ubze",
+      "name": "BeeZee",
+      "display": "bze",
+      "symbol": "BZE",
+      "logo_URIs": {
+	      "png": "https://raw.githubusercontent.com/bze-alphateam/Official-BZEdge-Graphics/master/Digital-Web/icon/Color/BZEdge_Color_icon.png"
```
Comment (JeremyParish69) [inline beezee/assetlist.json:24]: the issue is that the image URI is still pointing outside of this repo. It's should look more like: "https://raw.githubusercontent.com/cosmos/chain-registry/master/beezee/images/bze.png"

Comment (faneaatiku) [reply beezee/assetlist.json]: Gotcha! Thanks! changed it.

**Pull request merged**